server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -172,11 +172,14 @@ static TransportVersion def(int id) {
     public static final TransportVersion TIMEOUT_GET_PARAM_FOR_RESOLVE_CLUSTER = def(8_838_0_00);
     public static final TransportVersion INFERENCE_REQUEST_ADAPTIVE_RATE_LIMITING = def(8_839_0_00);
     public static final TransportVersion ML_INFERENCE_IBM_WATSONX_RERANK_ADDED = def(8_840_0_00);
+    public static final TransportVersion COHERE_BIT_EMBEDDING_TYPE_SUPPORT_ADDED_BACKPORT_8_X = def(8_840_0_01);
+    public static final TransportVersion REMOVE_ALL_APPLICABLE_SELECTOR_BACKPORT_8_X = def(8_840_0_02);
```

Comment:
I am not sure if these versions need to be added here because it is not valid upgrade path from `8.19` to `9.0`. Looking into it.

Reply:
Discussed offline, there should be no `8.x` versions in `9.0`.